include/revng/PipelineDescriptor/AnalysesList.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+/* TUPLE-TREE-YAML
+name: AnalysesList
```

Comment:
The `PipelineManager` API uses `AnalysesList` (this was discussed when the term was introduced). Since it's a list of `Analyses` (the plural form of `Analysis`)

Reply:
In English it's preferable to use "cat list" instead of "cats list" ([random source](https://www.proz.com/forum/linguistics/245800-noun_list_is_noun_singular_or_plural.html)) and that's the practice we adopt elsewhere too.

Anyway, if we already have other old code using plural, it makes sense to be coherent. But in the future let's try to follow this practice.